app/src/forms/agriseafoodopscreening/assets/generate-submission-pdf-001.html

```diff
@@ -173,39 +173,39 @@ <h2>Primary Contact</h2>
         <p class="Normal1">First Name</p>
       </td>
       <td style="width:248.25pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="331">
-        <p class="Normal1">{d.contacts[i].firstName}</p>
+        <p class="Normal1">{d.contacts[i, contactType = 'PRIMARY'].firstName}</p>
       </td>
     </tr>
     <tr>
       <td style="width:183.75pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="245">
         <p class="Normal1">Last Name</p>
       </td>
       <td style="width:248.25pt;padding:0cm 5.4pt 0cm 5.4pt" valign="top" width="331">
-        <p class="Normal1">{d.contacts[i].lastName}</p>
+        <p class="Normal1">{d.contacts[i, contactType = 'PRIMARY'].lastName}</p>
```

Comment:
so you can point to an item in an array by referencing more than one key?!

Reply:
Yeah, under [repetitions](https://carbone.io/documentation.html#repetitions), check out filters.  I wanted to do this before, but it was still a word document, and I think that maybe it was messing up the quotes and therefore not working. But works now and is a better way to do it.